config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
+  resources :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  post "/signup", to: "users#create"
+  post "/login", to: "sessions#create"
+  get "/authorized", to: "sessions#show"
+  # get "/dashboard", to: "users#show"
```

Comment:
[Optional] Since we don't use the `/dashboard` route, it's better to respect the YAGNI principle and remove it from the comment as well.